api/tech/signing/handlers/sigstore/attr/attr.go

```diff
@@ -14,6 +14,7 @@ const (
 )
 
 var defaultAttr = Attribute{
+	// https://github.com/tektoncd/chains/blob/main/docs/config.md#keyless-signing-with-fulcio
```

Comment:
What is the purpose of this link?

Reply:
that's the place where the fulcio defaults are defined - I thought